### DISCUSSION TITLE
Supported InfluxDB versions

### DISCUSSION THREAD

**Discussion by macminoz** [MarcusWichelmann/SolarEdgeExporter#5] [Q&A] [answered]
I'm looking to check which InfliuxDB version can be used with this awesome program. I'm running 1.x release at the moment, any compatibility? Thanks

**Comment by MarcusWichelmann** [comment, marked as answer]
Hi, I'm using InfluxDB 2.3 and that's also the only version I tested it with. Other v2 versions will probably work, but I don't know how things will look like with v1. I unfortunately cannot help you with that.
